.travis.yml

```diff
@@ -3,6 +3,8 @@ dist: trusty
 rust: nightly
 sudo: required
 
+cache: rust
```

Comment:
What does this do? All I can find are docs for `cache: cargo`.

Reply:
Oops, misremembered it :)